include/splinterdb/splinterdb.h

```diff
@@ -97,9 +97,14 @@ splinterdb_open(const splinterdb_config *cfg, splinterdb **kvs);
 
 // Close a splinterdb
 //
-// This will flush all data to disk and release all resources
+// This will flush all data to disk and release all resources.
+// Handle is NULL'ed out on a succcessful close.
 void
-splinterdb_close(splinterdb *kvs);
+splinterdb_close(splinterdb **kvs);
```

Comment:
This will be in /main once PR #396 passes review and merged to /main. (Will pull this out then.)

Reply:
(Have reverted these diffs, so that this change-set reduces to just the C-sample programs. Ignore this comment.)